### DISCUSSION TITLE
We can’t log you in. Please review your SSO settings.

### DISCUSSION THREAD

**Discussion by ThiagoMouro** [prisma/prisma#11219] [Q&A] [answered]
I need your help.

Authentication Failed!
We can’t log you in. Please review your SSO settings.

I used this:  https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/setup-sso-integration-on-prisma-cloud/setup-sso-integration-on-prisma-cloud-for-okta.html#:~:text=To%20secure%20administrator%20access%20to,complete%20the%20setup%20on%20Okta.

**Comment by janpio** [comment, marked as answer]
I think you are with the wrong Prisma. We are an Open Source ORM, you should try to talk to Palo Alto Networks who have a product of the same name.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
